datafusion/proto/build.rs

```diff
@@ -61,8 +61,9 @@ fn build() -> Result<(), String> {
     let json = std::fs::read_to_string(out.join("datafusion.serde.rs")).unwrap();
     let mut file = std::fs::OpenOptions::new()
         .write(true)
+        .truncate(true)
         .create(true)
-        .open("src/generated/datafusion_json.rs")
+        .open("src/generated/datafusion.rs")
```

Comment:
I was generating to two different files because it seemed like it was smooshing the code together when I switched feature sets. Now I realize I had to call `.truncate(true)` below.